runtime/src/weights/block_weights.rs

```diff
@@ -24,7 +24,7 @@ pub mod constants {
 	parameter_types! {
 		/// Importing a block with 0 Extrinsics.
 		pub const BlockExecutionWeight: Weight =
-			Weight::from_ref_time(constants::WEIGHT_REF_TIME_PER_NANOS.saturating_mul(5_000_000));
+			Weight::from_parts(constants::WEIGHT_REF_TIME_PER_NANOS.saturating_mul(5_000_000), 0);
```

Comment:
Is it valid to have a zero-proof size here? I think changing headers and other block generation details might have some size. @zktony Did you compare it with other runtimes?

Reply:
this is what latest version of template is using: [here](https://github.com/substrate-developer-hub/substrate-parachain-template/blob/ebcc7250a829e5374f3aa19834b840c460741fd8/runtime/src/weights/block_weights.rs#L27)

Reply:
Yes, other parachains are also utilizing the same value.